apps/rush-lib/src/cli/logic/pnpm/PnpmShrinkwrapFile.ts

```diff
@@ -275,4 +281,11 @@ export class PnpmShrinkwrapFile extends BaseShrinkwrapFile {
       return undefined;
     }
   }
+
+  protected checkValidVersionRange(dependencyVersion: string, versionRange: string) : boolean {
```

Comment:
>protected [](start = 2, length = 9)

Suggest to add a "// override" comment to clarify that we're overriding the bae class function.

Reply:
it. III

---
In reply to: [182983742](https://github.com/Microsoft/web-build-tools/pull/615#discussion_r182983742) [](ancestors = 182983742)